src/internal/components/dropdown/interfaces.ts

```diff
@@ -132,6 +132,15 @@ export interface DropdownProps extends ExpandToViewport {
    * Called when focus leaves the trigger or dropdown content.
    */
   onBlur?: NonCancelableEventHandler<Pick<React.FocusEvent, 'target' | 'relatedTarget'>>;
+
+  /**
+   * ID for the dropdown content wrapper
+   */
+  dropdownContentId?: string;
+  /**
+   * HTML role for the dropdown content wrapper
+   */
+  dropdownContentRole?: string;
```

Comment:
Would it make sense to "enum" this like we do `ariaHasPopup`?

Reply:
Given that there are so many options I'm not sure it makes so much sense here. We could _perhaps_ limit it to the options we think might be relevant here, but especially as it's an internal interface I'm not sure it has so much value

Reply:
Right, but `ariaHasPopup` is an internal prop as well, right?

And yes, I would limit it to the values that we currently think are reasonable (which I think is just `dialog`?).

Not a blocking comment, though. I'll leave the final call up to you.